specification/botservice/resource-manager/Microsoft.BotService/preview/2017-12-01/examples/ListTokensByBotService.json

```diff
@@ -0,0 +1,36 @@
+{
+    "parameters": {
```

Comment:
The file name says ListTokensByBotService...shouldn't it be ListConnectionsByBotService?